manifests/service.pp

```diff
@@ -14,12 +14,12 @@
 #
 # This class file is not called directly
 class nginx::service (
-  $service_restart = $nginx::service_restart,
-  $service_ensure  = $nginx::service_ensure,
-  $service_enable  = $nginx::service_enable,
-  $service_name    = $nginx::service_name,
-  $service_flags   = $nginx::service_flags,
-  $service_manage  = $nginx::service_manage,
+  Optional[String[1]] $service_restart                          = $nginx::service_restart,
+  Enum['running', 'absent', 'stopped', 'undef'] $service_ensure = $nginx::service_ensure,
```

Comment:
what's undef doing here?

Reply:
In https://github.com/voxpupuli/puppet-nginx/pull/1414 I chose `Stdlib::Ensure::Service`